operator/pkg/ingress/annotations/annotations_test.go

```diff
@@ -275,6 +275,105 @@ func TestGetAnnotationSSLPassthrough(t *testing.T) {
 	}
 }
 
+func TestGetAnnotationEnforceHTTPSEnabled(t *testing.T) {
+	type args struct {
+		ingress *networkingv1.Ingress
+	}
+
+	truePtr := new(bool)
```

Comment:
we should use `model.AddressOf...` in the tests too. https://github.com/cilium/cilium/blob/main/operator/pkg/model/helpers.go#L89